projects/components/src/navigation/nav-item/nav-item.component.ts

```diff
@@ -67,9 +67,15 @@ export class NavItemComponent {
 
     if (this.config.pageLevelTimeRangeIsEnabled && this.config.timeRangeResolver) {
       const timeRange = this.config.timeRangeResolver();
+      const timeRangeQueryParam = this.timeRangeService.toQueryParams(timeRange);
+      const timeRangeRefreshQueryParam = this.timeRangeService.getRefreshTimeRangeQueryParam();
```

Comment:
nit: could probably put this as a flag on the query param generation function rather than have two separate ones